canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-builder.scss

```diff
@@ -47,7 +47,12 @@ $operator-margin: 50%;
 	margin-bottom: 0;
 
 	.elyra-CodeMirror {
+		resize: vertical;
```

Comment:
Why do we need to set these stylings in both classes?

Reply:
Also, why do we need to set `!important`.  If there is a good reason please add a comment.  We should also add a comment as to why this styling was added.

Reply:
Sure @matthoward366 I have updated styling with comments. Please let me know if I need to change these.

I have included resize styling in elyra-CodeMirror which is the outer container so that the container grows along the text editor without this styling it is behaving like below.


https://github.com/user-attachments/assets/5cf65ef6-3001-4397-a196-79bda81be832

Similarly if I don't apply resize styling on inner `cm-editor` it would behave something like below where the editor doesn't resize.

https://github.com/user-attachments/assets/ed682fdd-5904-4115-93bd-7a9906f4226c